src/main/java/com/linecorp/armeria/common/metrics/MetricsData.java

```diff
@@ -0,0 +1,14 @@
+package com.linecorp.armeria.common.metrics;
+
+/**
+ * internal container for metric data
```

Comment:
Or... how about just duplicating this on the both sides given it's a fairly simple class. It's not very useful to a user at all.